src/core/channelz/channelz.h

```diff
@@ -128,6 +129,8 @@ class BaseNode : public RefCounted<BaseNode> {
  public:
   ~BaseNode() override;
 
+  void Orphaned() final {}
```

Comment:
Out of curiosity, why `final` here?  I wonder if there might be cases where a specific subclass may want to override this method.

Reply:
I *think* I want to make some optimizations later to de-virtualize this method, and so leaving it final for now leaves me that flexibility.